include/ofi.h

```diff
@@ -326,6 +326,16 @@ static inline void *ofi_get_page_end(const void *addr, size_t page_size)
 			+ page_size, page_size) - 1);
 }
 
+static inline bool ofi_is_size_aligned(size_t size, size_t alignment)
+{
+	return ((size % alignment) == 0);
+}
```

Comment:
You're using the word 'aligned' to mean 'multiple'.  In that regard, the function name is confusing.  ofi_is_size_multiple() is clearer to the intent, or at least based on the operation that is being performed.